### PR TITLE
Changes to ClusterServiceVersion

### PR DESCRIPTION
**Pull request by swghosh**
Changes to ClusterServiceVersion:
-  Include base64 encoded image icon, updated bundle description (from #88)
- OLM bundle: use "alpha" channel instead of "tech-preview"
    -  xref: http://external-ci-coldstorage.datahub.redhat.com/cvp/cvp-redhat-operator-bundle-image-validation-test/cert-manager-operator-bundle-container-v1.9.1-15/36c27912-933d-4136-a28c-667c2eb40311/operator-catalog-initialization-bundle-image-output.txt
    - in effect, OLM channel was updated in presubmit config: https://github.com/openshift/release/pull/35034

Co-authored-by: Thejas N <thn@redhat.com>
Signed-off-by: Swarup Ghosh <swghosh@redhat.com>

### CODE REVIEW

File: config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -42,11 +46,13 @@ spec:
       kind: CertManager
       name: certmanagers.operator.openshift.io
       version: v1alpha1
-  description: cert-manager-operator
-  displayName: cert-manager-operator
+  description: |
+    The cert-manager Operator for Red Hat OpenShift provides seamless support for cert-manager, which automates certificate management.
+    For more information, see [cert-manager.io](https://cert-manager.io).  displayName: cert-manager-operator
```
Comment (thejasn) [inline config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml:51]:
```suggestion
    The cert-manager Operator for Red Hat OpenShift provides seamless support for cert-manager, which automates certificate management.
    For more information, see [cert-manager.io](https://cert-manager.io).
```

File: bundle/manifests/cert-manager-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -4,8 +4,11 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
+    categories: Security
     olm.skipRange: <0.0.1
     operatorframework.io/suggested-namespace: cert-manager-operator
+    operators.openshift.io/valid-subscription: '["OpenShift Container Platform", "OpenShift
+      Platform Plus"]'
```
Comment (thejasn) [inline bundle/manifests/cert-manager-operator.clusterserviceversion.yaml:11]:
```suggestion
    operators.openshift.io/valid-subscription: '["OpenShift Container Platform", "OpenShift Platform Plus"]'
```

File: config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (swghosh) [reply config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml:51]: Updated.

File: bundle/manifests/cert-manager-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (swghosh) [reply bundle/manifests/cert-manager-operator.clusterserviceversion.yaml]: Updated.

**Comment by swghosh** [comment]
/cc @TrilokGeer 
PTAL

**Comment by swghosh** [comment]
/label docs-approved
/label px-approved
This PR doesn't introduce any user facing changes, only nits. In case, description needs an update from PX perspective, we can prepare a follow-up PR.

**Pull request merged**